target/scripts/helper-functions.sh

```diff
@@ -268,6 +268,17 @@ CHKSUM_FILE=/tmp/docker-mailserver-config-chksum
 # Compute checksums of monitored files.
 function _monitored_files_checksums
 {
+  # If a wildcard path pattern (or an empty ENV) would yield an invalid path
+  # or no results, `shopt -s nullglob` prevents it from being added.
+  shopt -s nullglob
+
+  # React to any cert changes within the following letsencrypt locations:
+  local DYNAMIC_FILES
+  for FILE in /etc/letsencrypt/live/"${SSL_DOMAIN}"/*.pem /etc/letsencrypt/live/"${HOSTNAME}"/*.pem /etc/letsencrypt/live/"${DOMAINNAME}"/*.pem
```

Comment:
This makes it more readable

```suggestion
  for FILE in /etc/letsencrypt/live/"${SSL_DOMAIN}"/*.pem \
              /etc/letsencrypt/live/"${HOSTNAME}"/*.pem   \
              /etc/letsencrypt/live/"${DOMAINNAME}"/*.pem
```